src/schematools/contrib/django/models.py

```diff
@@ -238,12 +245,11 @@ def name_from_schema(cls, schema: DatasetSchema) -> str:
     def create_for_schema(cls, schema: DatasetSchema) -> Dataset:
         """Create the schema based on the Amsterdam Schema JSON input"""
         name = cls.name_from_schema(schema)
-
         return cls.objects.create(
             name=name,
             schema_data=schema.json_data(),
             auth=_serialize_claims(schema),
-            url_prefix=schema.url_prefix,
```

Comment:
There's whole set of logic for `url_prefix` ;)
Like this one: https://github.com/Amsterdam/schema-tools/blob/0f499b3d422dfeaa66e90565c9e235efc9dcd288/src/schematools/utils.py#L83-L85